.github/workflows/build_linux.yml

```diff
@@ -98,19 +83,15 @@ jobs:
         run: |
           sudo apt install openssl
 
-      - name: Install Unity
-        uses: nick-invision/retry@v2
+      - id: unity_setup
+        uses: firebase/firebase-unity-sdk/gha/unity@unity-gha
         with:
-          timeout_minutes: 15
-          max_attempts: 3
-          shell: bash
-          command:  |
-            python scripts/gha/unity_installer.py --install --platforms ${{ inputs.unity_platform_name }} --version ${{ inputs.unity_version }}
+          version: ${{ inputs.unity_version }}
+          platforms: ${{ inputs.unity_platform_name }}
 
-      - name: Setup Unity path
+      - name: Setup Env
```

Comment:
nit: seems it's only setup swig related environment, so let's be a bit more specific, set the name to Setup Swig Env